jamfeed.com.jfsites.json

```diff
@@ -0,0 +1,26 @@
+{  
+   "providerId":"jamfeed.com",
+   "providerName":"Jamfeed",
+   "serviceId":"jf-site",
+   "serviceName":"Jamfeed Site",
+   "version": 1,
+   "syncPubKeyDomain": "jamfeed.com",
+   "warnPhishing": true,
```

Comment:
Not really needed if you sign the request. Setting it may lower your conversion as the users may abandon due to the warning.